services/drupal/web/themes/epa_theme/source/_patterns/05-components/box/box.twig

```diff
@@ -51,3 +51,11 @@
     </div>
   {% endif %}
 </div>
+
+{% if is_demo %}
```

Comment:
Moving forward, the `is_demo` variable should be replaced with the `show_documentation` variable when wrapping documentation.